pom.xml

```diff
@@ -43,6 +43,7 @@
     <module>elasticsearch</module>
     <module>web</module>
     <module>cli</module>
+    <module>benchling</module>
```

Comment:
The Benchling module should come before the web and CLI modules, so that it compiles first.